index.html

```diff
@@ -15,6 +15,7 @@
 <body>
```

Comment:
The layout of your app is not matching the Figma template you choose. Kindly implement the same layout as in the template including background images, margins, paddings, colors, font size, font family, etc. For this project, it is a must for students to follow the exact layout as in the Figma templates. Also, the portion marked as the cross is not part of the layout.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/185548745-8b9fef7f-8024-4d60-89f3-cf5e6f6c64ab.png) | ![image](https://user-images.githubusercontent.com/33268927/185548630-f1298a6b-e5b8-44e4-ad4e-8d942684b699.png) |

Reply:
- [ ] Kindly make sure to implement the changes asked by the previous reviewer [here](https://github.com/luccamicale/First-Mobile-Version-/pull/1#discussion_r949811470). Your design should be similar to figma design with following the same colour,background images, margins, paddings, font size, font family, etc

Reply:
- [ ] Kindly implement the changes asked in the attached loom video. Your design should be similar to Figma design with following the same colour, background images, margins, paddings, font size, font family, etc